client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1986,5 +1986,7 @@
   "warning.check-before-vaccinating": "Vaccination records from other stores may not be up to date. Ensure you have synced recently, and always confirm with the patient first.",
   "warning.field-not-parsed": "{{field}} not parsed",
   "warning.nothing-to-supply": "Nothing left to supply!",
-  "tooltip.defined-by-gs1-matrix": "Defined by GS1 datamatrix"
+  "tooltip.defined-by-gs1-matrix": "Defined by GS1 datamatrix",
+  "label.emergency": "Emergency",
+  "label.new-emergency": "New emergency requisitions"
```

Comment:
@jmbrunskill FYI have added a more verbose description on dashboard.

I thought this would make more sense because just having 'emergency' doesn't seem clear to me that it would also filter for new given that another category is explicitly called 'New'

Reply:
I've updated this to "label.new-emergency-requisitions", will help with translations if we make the key more explicit when we're using it for a specific purpose.